src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/integration/legacy/sentence/LegacyUpdateSentenceTests.kt

```diff
@@ -17,28 +18,6 @@ class LegacyUpdateSentenceTests : IntegrationTestBase() {
   @Autowired
   private lateinit var sentenceHistoryRepository: SentenceHistoryRepository
 
-  @Test
```

Comment:
this test has become obsolete because they cannot update the charge number from nomis